Configuration/Generator/python/Upsilon4swithBuToKstarJPsiToMuMu_forSTEAM_13TeV_TuneCUETP8M1_cfi.py

```diff
@@ -28,6 +28,7 @@
             'Bottomonium:O(3S1)[1S0(8)] = 0.02',
             'Bottomonium:O(3S1)[3P0(8)] = 0.02',
             'Bottomonium:gg2bbbar(3S1)[3S1(1)]g = on',
+            'Bottomonium:gg2bbbar(3S1)[3S1(1)]gm = on',
```

Comment:
hi @alberto-sanchez - could you make a small helper pSet that can then be imported by each of these configs to avoid having to make the same change 7 times (as I guess this change will also affect lots of config files outside of CMSSW...)